cmd/web/web.go

```diff
@@ -26,6 +36,22 @@ func NewWebCmd() *cobra.Command {
 	serveCmd.Flags().IntVarP(&port, "port", "p", 8080, "The port for the HTTP service to listen at")
 	serveCmd.Flags().StringVar(&araAddr, "ara-addr", "127.0.0.1:8000", "Address where ARA is running (ex: localhost:80)")
 
+	serveCmd.Flags().StringVar(&dbhost, "dbhost", "localhost", "The database host")
+	serveCmd.Flags().StringVar(&dbport, "dbport", "5432", "The database port to connect to")
```

Comment:
@nelsonkopliku maybe we should use db-port, db-host etc for consistency with ara-addr and the other flags

Reply:
Sure, makes sense to me